README.md

```diff
@@ -195,9 +195,13 @@ Or you can simply pull it from Jina Cloud:
 left_da = DocumentArray.pull('demo-leftda', show_progress=True)
 ```
 
+**Note**
+If you have at least XGB of RAM and want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.
```

Comment:
Should "XGB" be a specific number of gigabytes of RAM? "At least XGB" doesn't make sense to me.

Reply:
yes, I am trying to figure X out